src/main/java/sk/hfa/configuration/security/HfaSecurityConfiguration.java

```diff
@@ -85,6 +81,11 @@ protected void configure(HttpSecurity http) throws Exception {
         setCsrf(http);
     }
 
+    @Override
+    public void configure(WebSecurity web) throws Exception {
```

Comment:
Ked som to mergol ma upozornil intelij ze ta Exception sa nikdy nevyhodi tak asi je tam zbytocne